model/textparse/openmetricsparse.go

```diff
@@ -241,21 +241,21 @@ func (p *OpenMetricsParser) Next() (Entry, error) {
 	case tEOF:
 		return EntryInvalid, errors.New("data does not end with # EOF")
 	case tHelp, tType, tUnit:
-		switch t := p.nextToken(); t {
+		switch t2 := p.nextToken(); t2 {
 		case tMName:
 			p.offsets = append(p.offsets, p.l.start, p.l.i)
 		default:
-			return EntryInvalid, parseError("expected metric name after HELP", t)
+			return EntryInvalid, parseError("expected metric name after "+t.String(), t2)
 		}
-		switch t := p.nextToken(); t {
+		switch t2 := p.nextToken(); t2 {
 		case tText:
 			if len(p.l.buf()) > 1 {
 				p.text = p.l.buf()[1 : len(p.l.buf())-1]
 			} else {
 				p.text = []byte{}
 			}
 		default:
-			return EntryInvalid, parseError("expected text in HELP", t)
+			return EntryInvalid, fmt.Errorf("expected text in %s, got none", t.String())
```

Comment:
I would keep current wording, adding got none seems confusing to me.

Reply:
I see your point, but I don't think that `\"INVALID"\` is better. What this error is supposed to communicate is that there was no value for a metadata field, not that there was some invalid expression. If you still want to go with `\"INVALID"\` though, I respect your opinion.

Reply:
fmt.Errorf("expected text in %s"), without parseError. WDYT?

Reply:
cc @juliusv WDYT here?

Reply:
Sounds good to me.

Reply:
Sounds good to me as well.